web/pom.xml

```diff
@@ -138,7 +138,12 @@
       <dependency>
 		 <groupId>org.apache.velocity</groupId>
 		 <artifactId>velocity-tools</artifactId>
-	  </dependency>      
+      </dependency>
+      <dependency>
+          <groupId>com.sun.xml.ws</groupId>
+		 <artifactId>jaxws-rt</artifactId>
+		 <version>2.2.9-b14002</version>
+      </dependency>
```

Comment:
@rkorytkowski It is needed to throw UnsupportedMediaException(com.sun.xml.ws.server.UnsupportedMediaException)